tests/rust/pipe-test/pop_wait/client.rs

```diff
@@ -113,9 +118,9 @@ impl Drop for PipeClient {
     // Releases all resources allocated to a pipe client.
     fn drop(&mut self) {
         // Ignore error.
-        if let Err(e) = self.libos.close(self.pipeqd) {
-            println!("WARN: leaking pipeqd={:?}", self.pipeqd);
-            println!("ERROR: close() failed (error={:?})", e);
-        }
+        //     if let Err(e) = self.libos.close(self.pipeqd) {
+        //         warn!("leaking pipeqd={:?}", self.pipeqd);
+        //         error!("close() failed (error={:?})", e);
+        //     }
```

Comment:
Why remove this? If this is the right fix, could you remove it, instead of commenting them out?

Reply:
I've put it back. The close will fail because the pipe is full but I think it will be ok once PR #801 goes in.